ca/certificate-authority.go

```diff
@@ -298,6 +315,31 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		}
 	}
 
+	// Process requested extensions.  For now, the only extension we support is
+	// TLS Feature [RFC7633], i.e., "Must Staple", and for that we ignore the
+	// value provided by the client and just overwrite it with the specific value
+	// that only requires stapling.
+	//
+	// Other requested extensions are silently ignored.
+	requestedExtensions := []signer.Extension{}
+	for _, attr := range csr.Attributes {
+		if !attr.Type.Equal(oidExtensionRequest) {
+			continue
+		}
+
+		for _, extList := range attr.Value {
+			for _, ext := range extList {
+				if ext.Type.Equal(oidTLSFeature) {
```

Comment:
This doesn't appear to check whether the TLS Feature extension is actually requesting the mustStapleFeatureValue specifically.